roles/srv-announcer/tasks/install.yml

```diff
@@ -1,47 +1,23 @@
-- name: copying {{ srv_announcer_service_name }} artifact
-  copy:
-    src: "{{ sft_artifact_file_path }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
-  when:
-    - srv_announcer_artifact_file_path is defined
-    - srv_announcer_version is not defined
-    - srv_announcer_artifact_file_url is not defined
-    - srv_announcer_artifact_checksum is not defined
-
-- name: downloading {{ srv_announcer_service_name }} artifact by version
-  get_url:
-    url: "{{ srv_announcer_download_base_url }}/{{ srv_announcer_artifact_basename }}.tar.gz"
```

Comment:
Just a suggestion. But if we move this interpolatio-logic into `defaults` into a variable named `srv_announce_url:  {{ srv_announcer_download_base_url }}/{{ srv_announcer_artifact_basename }}.tar.gz`   doesn't that make both @lucendio and @flokli happy? as then both usecases are supported?  (Override the entire URI; or just override the version)

Reply:
If I understand you correctly, the proposal would basically be move the [deleted lines](https://github.com/wireapp/ansible-sft/pull/25/commits/c301c45d629e5c2b17ec24df6bc39cff01662eb8#diff-4e4a7fa6b5d4044be56da310925a8eaea2eb820d51c8d3abba9e8f258d0528e4L5-L6) over to `defaults/main.yml`. For that, the [checksum list](https://github.com/wireapp/ansible-sft/pull/25/commits/c301c45d629e5c2b17ec24df6bc39cff01662eb8#diff-4e4a7fa6b5d4044be56da310925a8eaea2eb820d51c8d3abba9e8f258d0528e4L17-L21) still needs to be maintained, which is - and that would be my guess - what @ffflorian wanted to get rid of. While I personally think it would be a valuable user experience to only define the version and don't care about anything else, I can see the overhead in this.

My intention with the last commit was basically to fix this PR and move on. But I'm also happy to take a step back again and discuss the three ways of specifying the artifact that is being used.

Reply:
I think having the hashes is fine.  Lets keep it.   If things turn out to be needed to be more override-able  we can always send a patch later. Let me approve this PR so you can merge it

Reply:
I thought the hashes part was kept in from the discussion. I didn't look at the diff. I think  I approved too soon now. Let me open a PR that reintroduces that part. sorry for the noise